src/python/tests/CMakeLists.txt

```diff
@@ -38,10 +38,19 @@ set(python_tests_robot
   testAmazonPod.py
 )
 
-# these tests use Atlas v5 robot model
+# Tests are run by default on Valkyrie (val2)
+#
+# If intending to test Atlas only, please set _extra_args -v5
+# Tests can also be run for both Atlas and Valkyrie by setting an additional
+#  _extra_args_2 -v5
+
+# these tests are tested on Atlas v5 robot model ONLY
```

Comment:
Maybe document how [test]_extra_args and [test]_extra_args_[n] works, but don't comment about Atlas and Valkyrie options, those seem self explanatory once you understand how extra_args works.